### PR TITLE
Updating staged release pipeline

### PR DESCRIPTION
**Pull request by JackQuincy**
Updating the prod-release.yml to do a staged release across all regions.

### CODE REVIEW

File: .pipelines/prod-release.yml [hunk 1]
```diff
@@ -3,16 +3,69 @@ pr: none
...
-  - template: ./templates/template-job-deploy-azure-region.yml
+  - template: ./templates/template-job-deploy-azure-env.yml
     parameters:
-      environment: RP-Prod
+      environment: RP-Prod-USSector
       locations:
       - eastus
       - eastus2
+      - southcentralus
+      - westus2
```
Comment (jim-minter) [inline .pipelines/prod-release.yml:33]: will need to find a way to disable individual e2es where quota is not available

Comment (jim-minter) [reply .pipelines/prod-release.yml:33]: seems like westus2 is the only one in this case

Comment (JackQuincy) [reply .pipelines/prod-release.yml:34]: As opposed to skipping the e2es I'm switching to allowing running the next stage even if the previous one failed

Comment (jim-minter) [reply .pipelines/prod-release.yml:34]: perfect

**Pull request merged**